.github/scripts/lint-pr-title.cjs

```diff
@@ -0,0 +1,106 @@
+/**
+ * This Script can be tested locally with
+ *
+ * export GH_TOKEN=**** GITHUB_REPOSITORY=quisquous/cactbot PR_NUMBER=$NUM
+ * node ./.github/scripts/lint-pr-title.cjs
+ */
+'use strict';
+
+const github = require('@actions/github');
+
+const validScope = [
+  'resources',
+  'ui',
+  'pullcounter',
+  'raidboss',
+  'raidemulater',
+  'oopsy',
+  'radar',
+  'eureka',
+  'jobs',
+  'test',
+  'util',
+  'config',
+  'lint',
+  'build',
+  'ci',
+];
+
+const getComment = (title, userName) => `${userName ? '@' + userName : ''} Thanks for your contribution!🚀
+
+Currently your title is: ${title},
+but it should be in the format of \`scope: description\`.
+
+\`scope\` can be any of the following:
+${validScope.map((s) => `  - ${s}`).join('\n')}
+
+------
+This comment is created and updated by a bot.
+`;
+
+/**
+ * @param {ReturnType<typeof import("@actions/github").getOctokit>} octokit
+ * @param {string} owner
+ * @param {string} repo
+ * @param {number} pullNumber
+ * @return {Promise<boolean>}
+ */
+const checkTitle = async (octokit, owner, repo, pullNumber) => {
+  const { data: pullRequest } = await octokit.rest.pulls.get({
+    owner,
+    repo,
+    'pull_number': pullNumber,
+  });
+  const { title, user } = pullRequest;
+  const userName = user?.login;
+  const m = /^(?:\!|\[WIP\])?(?<scope>\w+):\s?.+$/.test(title);
```

Comment:
- The `!` stands for `BREAKING CHANGES`, which is described in Conventional Commit.
- What do you mean "valid non-scope prefixes", can you list some of them?
- Maybe adding `i` flag to regex matching makes sense.
- Yeah, nice idea.

Reply:
* Oh, ok `!` seems find then.
* I meant `[WIP]` or `[wip]` or `!` as a valid non-scope prefix, in other words something that goes in front of the scope that is not a scope but is also valid
* I think I would argue against `i` flag, because people are already inconsistent about `raidboss` vs `Raidboss` etc, and I think it'd be better to be consistent here (to make my life easier with changelogs).  `[WIP]` vs `[wip]` doesn't bother me because that will never get merged.  If you only want to have `[WIP]` that's fine too, I don't feel strongly that we also need `[wip]` just that I sometimes use that.